Ratings_MVP/Ratings_MVP/Services/NetworkService/RequestType.swift

```diff
@@ -0,0 +1,22 @@
+// RequestType.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Type of requests
+enum RequestType {
```

Comment:
```suggestion
enum MovieType {
```
вдруг реквесты будут не только такими)